packages/utilities/src/appendFunction.ts

```diff
@@ -0,0 +1,19 @@
+// tslint:disable:no-any
+
+/**
+ * Returns a single function which will call each of the given functions in the context of the
+ * parent.
+ */
+export function appendFunction<TRetVal>(parent: any, ...functions: (null | undefined | any)[]): () => TRetVal {
```

Comment:
`any` is the same as `null | undefined | any`. Can you use `Function` instead of `any` here? What happens if `parent` is undefined or null?

Reply:
I really, really tried... I could try again.

Reply:
I can take a stab at it too. :)